usdt-impl/src/common.rs

```diff
@@ -91,13 +110,21 @@ fn unpack_argument_lambda(types: &[dtrace_parser::DataType]) -> TokenStream {
     }
 }
 
-// Convert a supported data type to one passed to the probe function in a register
-fn asm_type_convert(typ: &dtrace_parser::DataType, input: TokenStream) -> TokenStream {
+// Convert a supported data type to a type to store for the duration of the
+// probe invocation and 2. a transformation for compatibility with an asm
+// register.
```

Comment:
A `1.` before the "type to store..." would be helpful, so the `2.` doesn't come out of left field :)

Reply:
yeesh.. not sure how I missed that